### PR TITLE
Cu 2rjuuaj

### PR DESCRIPTION
**Pull request by RealDavidH**
Updated desktop header to reflect the design on figma.

I could not update the mobile header

### CODE REVIEW

**Comment by abstudios617** [comment]
Task linked: [CU-2rjuuaj Update Header FE](https://app.clickup.com/t/2rjuuaj)

**Review by AlamTahmidul — CHANGES_REQUESTED** [review]
- In the `client/package.json` be sure to put the `jest` unit-test back in
- In the `index.html`, uncomment the chat pop-up

File: client/src/components/nav/nav.styles.js [hunk 1]
```diff
@@ -1,49 +1,107 @@
...
+    height: auto;
+    width: 100%;
   }
+`;
 
+// Start of nav top
+export const NavTop = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
```
Comment (callmejim1226) [inline client/src/components/nav/nav.styles.js:18]: Just a tiny detail for following the mock-up design, but to just give a space between the top and bottom nav and between top page and top nav, I suggest adding a line, 'padding: 15px 0px'.

**Review by callmejim1226 — APPROVED** [review]
Looks good to me!

**Review by AlamTahmidul — APPROVED** [review]
Resolved

**Pull request merged**